packages/fxa-content-server/app/styles/tailwind/inputs.css

```diff
@@ -2,6 +2,57 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+@layer utilities {
+  /* For number inputs where we want to hide the step/increment arrows */
+  .no-step {
+    -moz-appearance: textfield;
+
+    &::-webkit-outer-spin-button &::::-webkit-inner-spin-button {
```

Comment:
Q: should this have four `::::`?

Reply:
Ah nope, not sure how that happened, thanks for the catch. 😁